### ISSUE TITLE
vertical alignment of checkboxes and radios is incorrect

### ISSUE THREAD

**Issue by garrett** [cockpit-project/cockpit#9806]
Cockpit has had issues with alignment of various widgets; most notably checkboxes and radios.

This has been made more prominent with #9166, #9681, #9791, & #9465.

I've investigated the problem and it seems like PatternFly has this issue, and it's actually a problem in Bootstrap itself (which PatternFly pulls in), as it makes assumptions for a specific browsers on a specific OS on a specific version from a while back. (I'm not sure which browser/OS/version combo.)

I discovered a fix that does the right thing for all browsers on all OSes, instead of hardcoding pixel values. The fix is in the PF issue @ https://github.com/patternfly/patternfly/issues/1122 — but I'd like to change this into a PR for both PatternFly and a 1-off backport to Cockpit, so we can benefit immediately.

**Comment by martinpitt** [comment]
All four referenced PRs got merged a while ago. Do we still have pages where this is the case?

**Comment by garrett** [comment]
Yep, this is a problem everywhere in Cockpit that doesn't use `ct-form-layout` (where I fixed it, among other issues, opt-in). Some places, it's more pronounced than others — it's more noticeable especially when more checkboxes and radio buttons are used.

It's an issue with Bootstrap and PatternFly inherited it. See the upstream issue. I think it hasn't gotten any traction as the whole PatternFly team (and probably a lot of Bootstrap devs) are all using Macs, where the widgets are different. If the widgets are sized with pixels and the widgets are different sizes across different browsers & OSes, some are going to be wrong.

Meanwhile, the code in the first three of those PRs have all been ported over to `ct-form-layout`, at least fixing it for the VM-related dialogs.

What's special about `ct-form-layout`'s checkbox and radio alignment? I used CSS to lay out things without hardcoding some magic pixel value that only works on a Mac. And then I checked in across several browsers and OSes to make sure it worked. Seems obvious? :man_shrugging:
